trafficcloud/app_config.py

```diff
@@ -83,4 +83,44 @@ def check_project_cfg_section(section):
     if section in cfp.sections():  # If the given section exists,
         return True                # then return True.
     else:                          # Otherwise,
-        return False               # then return False
\ No newline at end of file
+        return False               # then return False
+
+def update_tracking_config(config_path, update_dict):
+    """helper function to edit cfg files that look like run_tracking.cfg
+
+    update_dict: i.e. {'nframes': 10, 'video-filename': 'video.avi'}
+    """
+    with open(config_path, 'r') as rf:
+        lines = rf.readlines()
+    with open(config_path, 'w') as wf:
+        for line in lines:
+            line_updated = False
+            line_param = line.split('=')[0].strip()
+            if line_param in update_dict.keys():
+                wf.write("{} = {}\n".format(line_param, update_dict[line_param]))
+                line_updated = True
+            if not line_updated:
+                wf.write(line)
+
+def get_tracking_config(config_path, params_to_get):
```

Comment:
Any reason this function takes `params_to_get` instead of returning a whole dictionary of the config file? The second seems easier to implement and would turn this:

```
dict = get_tracking_config(config, ['hi', 'hello'])
return dict['hi']
```

to:

```
dict = get_tracking_config(config)
return dict['hi']
```